Objects/longobject.c

```diff
@@ -1935,6 +1935,7 @@ long_to_decimal_string_internal(PyObject *aa,
     } while (0)
 
     /* fill the string right-to-left */
+    assert (kind > 0);
```

Comment:
This assert would seem to fail when just `bytes_writer` is passed into the function.  It seems that this is the case where `kind` is uninitialized (the if-block at lines 1875-1896, particularly 1882-1887).

Reply:
We can fix that and land that PR first.